src/main/java/com/vmware/vipclient/i18n/VIPCfg.java

```diff
@@ -117,17 +125,46 @@ public void initialize(String vipServer, String productName, String version) {
         this.version = version;
         this.vipServer = vipServer;
     }
-
+    
+    /**
+     * Load client configuration from a JSON file in the resource folder 
+     * 
+     * @param configFile This is the name of the JSON configuration file 
+     * @throws ParseException 
+     * @throws IOException 
+     */
+    public void loadConfig(String configFile) throws VIPClientInitException {
+    	ClassLoader classloader = Thread.currentThread().getContextClassLoader();
+    	InputStream is = classloader.getResourceAsStream(configFile + ".json");
+    	
+    	Reader reader = new InputStreamReader(is);
+    	JSONParser parser = new JSONParser();
+		JSONObject jsonObject = null;
+		try {
+			jsonObject = (JSONObject) parser.parse(reader);
+			this.vipServer = (String) jsonObject.get("online_service_url");
```

Comment:
should validate 'online_service_url'? other parameters as well?

Reply:
Yes, we should do validation for the paramaters. I will add it.
 
Yes, it should read other parameters as well. Let's add them one by one, as we start using them. For example, I am about to add the "prod_mode" parameter because I need to use it for the exception/messaging logic.